### DISCUSSION TITLE
Images BUG

### DISCUSSION THREAD

**Discussion by powen56** [dompdf/dompdf#3481] [Q&A]
hello everyone, asking if anyone experience when exporting data with images does sometimes image not loading? or image not found.

but when exporting again it shows the images.

how can i improve for this one?

**Comment by bsweeney** [comment]
I don't think I've seen this before. Which version of Dompdf are you using? Is this a generated image or static one? Is Dompdf in a long-running process or instantiated new with each render? Happens only under heavy use or any time?
